src/pkgmt/modified.py

```diff
@@ -0,0 +1,47 @@
+import subprocess
+import argparse
+
+
+def check_modified(base_branch, exclude_path, debug=False):
+    # https://stackoverflow.com/questions/4380945
+    cmd = f"git diff --exit-code {base_branch}... -- ."
+
```

Comment:
also, I thin let's rename this to `fail_if_modified` to imply that if anything has been modified (excluding the --exclude-path), this will fail, aka return exit code 1